src/gfx/context.h

```diff
@@ -10,6 +10,7 @@ typedef struct GFX_Context GFX_Context;
 
 void GFX_Context_Attach(HWND hwnd);
 void GFX_Context_Detach();
+void GFX_Context_SetVsync();
```

Comment:
Can we change the signature to `GFX_Context_SetVsync(bool x)` to decouple GFX from Config?

Reply:
Ya idk what I was thinking when I wrote that lol. Fixed.